### PR TITLE
Brazil Para parser

### PR DESCRIPTION
**Pull request by AnyaLindstromBattle**
First look at Para parser. Feedback welcome.

### CODE REVIEW

File: ingestion/functions/parsing/brazil_para/para.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    "doenças autoimunes": "autoimmune disease"
+}
+
+_NONE_TYPES = set(["Não", "null", "undefined", None])
+
+_SYMPTOMS_MAP = {
+    "Dor de Cabeça": "headache",
+    "Distúrbios Gustativos": "taste alteration",
+    # Symptom ontology does not have a specific term for smell alterations
+    "Distúrbios Olfativos": "disturbances of sensation of smell and taste",
```
Comment (iamleeg) [inline ingestion/functions/parsing/brazil_para/para.py:55]: @Mougk should we add this term to the ontology, and is this phrasing OK?

**Review by iamleeg — COMMENTED** [review]
Looks good, just want to get answer from @Mougk about symptom ontology before approving.

**Comment by Mougk** [comment]
Does loss of smell exist? 

In any case it should be added because it is a common symptom of COVID-19

**Comment by AnyaLindstromBattle** [comment]
Loss of smell does not exist. In any case Distúrbios Olfativos translates to disturbances to sense of smell rather than specifically a loss of smell. So what I could do is add 'smell alteration'?

**Comment by Mougk** [comment]
Sounds good. Can we also add 'loss of smell' in the dictionary?

**Comment by AnyaLindstromBattle** [comment]
@Mougk In this case I won't add it to the dictionary as the data doesn't specifically report a 'loss of smell'. But in future if a dataset does I will make sure to.

**Review by fojackson8 — APPROVED** [review]
Yep all looks good

**Pull request merged**